pkg/server/routes.go

```diff
@@ -198,6 +200,15 @@ func (s *Server) routes(
 
 	// gql.Use(requirePrincipalMiddleware)
 
+	osConfig := opensearch.Config{
+		Addresses: []string{s.Config.GetString(appconfig.OpenSearchHostKey)},
+	}
+
+	osClient, err := opensearch.NewClient(osConfig)
+	if err != nil {
+		s.logger.Fatal("Failed to create an OpenSearch client", zap.Error(err))
```

Comment:
We should probably either
- Make a feature flag that prevents the creation/use of the OpenSearch client
- Change this log to not be `Fatal`

because when we push this code to our deployed envs, right now this will likely fail.